loading_NYC_taxi_data.ipynb

```diff
@@ -0,0 +1,854 @@
+{
```

Comment:
<p>This is fairly large output. Why not use <code>.head()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/harrislam1/covid19_NYC_taxi_trips/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>